docs/tendermint-core/configuration.md

```diff
@@ -189,6 +189,9 @@ size = 5000
 # size of the cache (used to filter transactions we saw earlier)
```

Comment:
Maybe we can be precise if size is in bytes or in transactions.

Reply:
yep, added `in transactions` suffix